Traceback (most recent call last):
  File "main.py", line 7, in <module>
    ray.init()
  File "/home/pargupta/.local/lib/python3.7/site-packages/ray/worker.py", line 700, in init
    ray_params=ray_params)
  File "/home/pargupta/.local/lib/python3.7/site-packages/ray/node.py", line 144, in __init__
    self._init_temp(redis_client)
  File "/home/pargupta/.local/lib/python3.7/site-packages/ray/node.py", line 243, in _init_temp
    try_to_create_directory(self._session_dir)
  File "/home/pargupta/.local/lib/python3.7/site-packages/ray/utils.py", line 856, in try_to_create_directory
    os.makedirs(directory_path, exist_ok=True)
  File "/cm/shared/apps/python/3.7.4-1910/lib/python3.7/os.py", line 221, in makedirs
    mkdir(name, mode)
OSError: [Errno 28] No space left on device: '/tmp/ray/session_2020-11-06_19-28-08_205631_24385'
srun: error: node099: task 0: Exited with exit code 1
